FunctionSignatureMap.php

```diff
@@ -8887,7 +8886,7 @@
 'pcntl_get_last_error' => ['int'],
 'pcntl_getpriority' => ['int', 'pid='=>'int', 'process_identifier='=>'int'],
 'pcntl_setpriority' => ['bool', 'priority'=>'int', 'pid='=>'int', 'process_identifier='=>'int'],
-'pcntl_signal' => ['bool', 'signo'=>'int', 'handle'=>'callable():void|callable(int):void|callable(int,array):void|int', 'restart_syscalls='=>'bool'],
+'pcntl_signal' => ['bool', 'signo'=>'int', 'handle'=>'callable(mixed...):?int', 'restart_syscalls='=>'bool'],
```

Comment:
I undid this on Phan's version; phan doesn't support that callable syntax and might end up with a different syntax